federation/cluster/federation-up.sh

```diff
@@ -90,10 +90,9 @@ function init() {
       --dns-zone-name="${DNS_ZONE_NAME}" \
       --dns-provider="${DNS_PROVIDER}" \
       --image="${kube_registry}/hyperkube-amd64:${kube_version}" \
-      --apiserver-arg-overrides="--storage-backend=etcd2" \
       --apiserver-enable-basic-auth=true \
       --apiserver-enable-token-auth=true \
-      --apiserver-arg-overrides="--v=4" \
+      --apiserver-arg-overrides="--runtime-config=api/all=true,--v=4" \
```

Comment:
any reason why --storage-backend=etcd2 is removed ?
Is it causing some problems?
In the absence of this, etcd3 will be used.
As far as I remember, this was intentional until there is a proper plan to move federation API server to etcd3, @madhusudancs, @shashidharatd.